pkg/ctl/create/nodegroup.go

```diff
@@ -44,7 +44,7 @@ func createNodeGroupCmdWithRunFunc(cmd *cmdutils.Cmd, runFunc func(cmd *cmdutils
 
 	cfg.Metadata.Version = "auto"
 
-	cmd.SetDescription("nodegroup", "Create a nodegroup", "", "ng")
```

Comment:
I don't think the warning should be shown here, because it's about a very specific option.

Reply:
Please also mention release `0.33.0` as the deprecation point.